### PR TITLE
RAC: `<Headline>` as React Aria Component

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

File: packages/components/src/Headline/Headline.tsx [hunk 1]
```diff
@@ -9,31 +10,23 @@ import {
...
-// ---------------
-export const Headline = ({
-  children,
+const _Headline = ({
   variant,
   size,
+  children,
   align = 'left',
   color,
-  level = '1',
```
Comment (sebald) [inline packages/components/src/Headline/Headline.tsx:35]: Hm. should we allow both so this is not a breaking change?

**Review by sebald — CHANGES_REQUESTED** [review]

File: packages/components/src/Headline/Headline.tsx (hunk 1 above)
Comment (sarahgm) [reply packages/components/src/Headline/Headline.tsx:35]: But than I can't use the prop from RAC oder?👀 @sebald

Comment (sebald) [reply packages/components/src/Headline/Headline.tsx:35]: You can cast it with `Number(level)`. If we only allow 1-6 as string plug 1-6 as number this should be safe :)

Comment (sarahgm) [reply packages/components/src/Headline/Headline.tsx:35]: done please have a loook :)

**Review by sebald — APPROVED** [review]

**Comment by sarahgm** [comment]
Why did you changed it? :( @sebald

**Comment by sebald** [comment]
@sarahgm if we are only using string, there is no good autocomplete and the cast to number might not work :)

**Comment by sarahgm** [comment]
I wrote a test for it... also i ask earlier that I want to use the level from react aria components. But yes your right. Just feels a bit not so good @sebald

**Pull request merged**